Aboutself.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+     <meta charset="UTF-8" >
+     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+     <link rel="stylesheet" href="aboutself.css">
+     <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+     <title>Mobile Version Website</title>
+  </head>
+    <body>
+      <div class="About">
+<h2 class="Abouth2">About
+    Myself</h2>
+    <section>
+        <p class="Hello">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </section>
+    <section>
+        <h3 class="Lets">Let's Connect</h3>
+    </section>
+    <section class="fonticons">
+          <ul>
+            <li><a href="#"><i class='fab fa-github'></i></a></li>
+            <li><a href="#"><i class='fab fa-linkedin-in'></i></a></li>
+            <li><a href="#"><i class='fab fa-angellist'></i></a></li> 
+            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#"><i class='fab fa-medium-m'></i></a></li>
+             </ul>
+          </section>
```

Comment:
Kindly note that you should only have one html file and one css file. Currently you have the about as a different page. It should make up a section in the page you did the headline in.
The structure of your page should be
`index.html`
`<html>`
`<body>`
`</header><nav>... </nav> </header>`

`<section>headline <section>`

`<section>works<section>`

`<section>About <section>`

`</body>`
`</html>`

Refer back to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)
<img width="1660" alt="Screenshot 2022-10-29 at 10 36 44" src="https://user-images.githubusercontent.com/48546992/198820046-1b439a78-34b1-473c-acec-d9b5ad245b6f.png">